tests/common/gllayerrenderer.cpp

```diff
@@ -81,6 +81,8 @@ void GLLayerRenderer::Draw(int64_t* pfence) {
 
   glDrawFrame();
 
+  int64_t gpu_fence_fd = -1;
```

Comment:
Hmm I actually think we should get of this logic since HWC should provide the synchronization.